src/components/map/layers/ThematicLayer.js

```diff
@@ -168,7 +168,10 @@ class ThematicLayer extends Layer {
                 <div>{indicator}</div>
                 <div>{periodName}</div>
                 <div>
-                    {i18n.t('Value')}: {value ? value : i18n.t('No data')}
+                    {i18n.t('Value')}:{' '}
+                    {value !== null && value !== undefined
```

Comment:
Is this different than the fix in master bc of babel support?

Reply:
Yes, I mentioned it in the description.